src/plugins/vis_default_editor/public/components/controls/raw_json.tsx

```diff
@@ -79,7 +79,7 @@ function RawJsonParamEditor({
       onBlur={setTouched}
     >
       <>
-        <CodeEditor
+        <CodeEditor // TODO Unsure how to resolve context. Requires CODEOWNER input.
```

Comment:
This also works fine now
<img width="652" alt="image" src="https://github.com/elastic/kibana/assets/17003240/29622ed2-9021-4618-bfe9-5e2a37908574">